packages/flutter/lib/src/painting/borders.dart

```diff
@@ -151,20 +132,48 @@ class BorderSide {
   /// A hairline black border that is not rendered.
   static const BorderSide none = BorderSide(width: 0.0, style: BorderStyle.none);
 
-  /// The direction of where the border will be drawn relative to the container.
-  final StrokeAlign strokeAlign;
+  /// The relative position of the stroke on a [BorderSide] in an
+  /// [OutlinedBorder] or [Border].
+  ///
+  /// Values typically range from -1.0 (inside border, default) to
+  /// 1.0 (outside border), without any bound constraints (e.g., a
+  /// value of -2.0 is is not typical, but allowed).
```

Comment:
I'd add mentions of the constants here, so the reader knows what they correspond to.

```suggestion
  /// Values typically range from -1.0 ([strokeAlignInside], inside border, default) to
  /// 1.0 ([strokeAlignOutside] outside border), without any bound constraints (e.g., a
  /// value of -2.0 is is not typical, but allowed). A value of 0 ([strokeAlignCenter]) will
  /// center the border on the edge of the widget.
```